src/BinSkim.Rules/BinarySkimmer.cs

```diff
@@ -28,11 +28,11 @@ public BinarySkimmer()
             {
                 this.SetProperty<string>(BinScopeCompatibility.EquivalentBinScopeRulePropertyName, altId);
             }
-        }
 
-        // BA2012.DoNotModifyStackProtectionCookie is the only rule
-        // that potentially fires warnings (and its default is error).
-        public override FailureLevel DefaultLevel => FailureLevel.Error;
+            // BA2012.DoNotModifyStackProtectionCookie is the only rule
+            // that potentially fires warnings (and its default is error).
+            this.DefaultConfiguration.Level = FailureLevel.Error;

```

Comment:
>FailureLevel.Error [](start = 46, length = 18)

FYI - this code is changed in BA2004, so we need to watch the merge carefully. The default needs to be warning for now, can explain further.

Reply:
I saw the change. i will do a manually merge and change to warning just like you did :)

---
In reply to: [551059054](https://github.com/microsoft/binskim/pull/325#discussion_r551059054) [](ancestors = 551059054)